app/assets/favicons/manifest.json.erb

```diff
@@ -10,5 +10,11 @@
 	"start_url": "/",
 	"theme_color": "#7ebc6f",
 	"background_color": "#fff",
-	"display": "minimal-ui"
+	"display": "minimal-ui",
+	"protocol_handlers": [
+		{
+			"protocol": "geo",
+			"url": "<%= Settings.server_protocol + "://" + Settings.server_url %>?geouri=%s"
```

Comment:
Is `<%= Settings.server_protocol + "://" + Settings.server_url %>` required? I don't see url protocols/hosts in examples [1](https://developer.chrome.com/docs/web-platform/best-practices/url-protocol-handler) [2](https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps/Manifest/Reference/protocol_handlers)